pkgs/applications/virtualization/lima/default.nix

```diff
@@ -39,10 +39,19 @@ buildGoModule rec {
 
   buildPhase = ''
     runHook preBuild
-    make "VERSION=v${version}" binaries
+    make "VERSION=v${version}" "CC=${stdenv.cc.targetPrefix}cc" binaries
     runHook postBuild
   '';
 
+  preCheck =
+    if stdenv.hostPlatform.isLinux then
+      ''
+        # Workaround for: could not create "/homeless-shelter/.lima/_config" directory: mkdir /homeless-shelter: permission denied
```

Comment:
I couldn't reproduce this exact failure, but I can confirm that LIMA_HOME has to be set to generate completion files, which makes the build fail as:

```
FATA[0000] no such file or directory                    
FATA[0000] no such file or directory                    
FATA[0000] no such file or directory                    
installShellCompletion: installed shell completion file `/nix/store/anvbywrxhwdb92q6gv1x27za5n1yax83-lima-1.0.1/share/bash-completion/completions/limactl.bash' does not exist or has zero size
```